src/helpers.js

```diff
@@ -111,12 +117,13 @@ export async function promptUser(packageName: string, dependencies: Array<Depend
       }, {
         text: 'Never',
         onDidClick: () => {
-          configFile.append('ignored', packageName)
+          ignoredPackages.push(packageName)
+          atom.config.set('sb-package-deps.ignored', ignoredPackages)
           if (!shownStorageInfo) {
             shownStorageInfo = true
             atom.notifications.addInfo('How to reset package-deps memory', {
               dismissable: true,
-              description: `If you ever wish to change the packages package-deps never installs, please modify ${tildify(configPath)}`,
+              description: `If you ever wish to change the packages package-deps never installs, please modify ${tildify(configPath)}.sb-package-deps.ignored`,
```

Comment:
Looks like `atom.config.getUserConfigPath()` actually returns the path to the `condif.cson` file, but putting a `.name.setting` directly on the end of the path doesn't make any sense.